man/systemd.unit.xml

```diff
@@ -1235,10 +1235,24 @@
         <varlistentry>
           <term><varname>ConditionFirmware=</varname></term>
 
-          <listitem><para>Check whether the system's firmware is of a certain type. Possible values are:
-          <literal>uefi</literal> (for systems with EFI),
-          <literal>device-tree</literal> (for systems with a device tree) and
-          <literal>device-tree-compatible(xyz)</literal> (for systems with a device tree that is compatible to <literal>xyz</literal>).</para>
+          <listitem><para>Check whether the system's firmware is of a certain type. Multiple values are possible.</para>
+
+          <para><literal>uefi</literal> for systems with EFI.</para>
+
+          <para><literal>device-tree</literal> for systems with a device tree.</para>
+
+          <para><literal>device-tree-compatible(<replaceable>value</replaceable>)</literal> for systems with a device tree that is compatible to
+          <literal>value</literal>.</para>
+
+          <para><literal>smbios-field(<replaceable>field</replaceable> <replaceable>operator</replaceable> <replaceable>value</replaceable>)</literal>
+          for systems with a SMBIOS field containing a certain value.
+          <literal>field</literal> is the name of the SMBIOS field which is read from
+          <filename>/sys/class/dmi/id/<replaceable>field</replaceable></filename>.
```

Comment:
unfortunately the sysfs attribute files are actually named differently from the fields in the SMBIOS spec. Hence maybe say `is the name of the SMBIOS field as exposed as <literal>sysfs</literal> attribute file below <filename>/sys/class/dmi/id/</filename>`

Reply:
Adjusted in latest version